master/pom.xml

```diff
@@ -742,7 +729,7 @@
       <dependency>
         <groupId>commons-logging</groupId>
         <artifactId>commons-logging</artifactId>
-        <version>[1.0,1.2]</version>
```

Comment:
this probably should be be a range as its not really deploying this library since slf4j is managing, and you never know about transitive versions.

Reply:
Done